packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -11,12 +11,22 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-if (!ShadowRoot.prototype.createElement) {
+if (!(ShadowRoot.prototype.createElement && ShadowRoot.prototype.importNode)) {
```

Comment:
Should this feature check be here at all?

Reply:
Reverted for now. Will have more to change/discuss to potentially better support Chrome's partial implementation in a follow-on.

Reply:
Based on dicussion with @justinfagnani and suggesting in #549, check was removed.